qutebrowser/browser/hints.py

```diff
@@ -748,6 +748,38 @@ def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
         self._context.tab.find_all_elements(selector, self._start_cb,
                                             only_visible=True)
 
+    def _get_visible_hints(self):
+        """Get elements which are currently visible.
+        """
```

Comment:
FWIW pylint has a [docstyle extension](https://github.com/PyCQA/pylint/blob/master/pylint/extensions/docstyle.py) which I turned on, but it's still quite rudimentary.

flake8 on this PR should be fine after you fix this docstring and the one for `_handle_auto_follow` FWIW, but Travis will hopefully correctly test it again after rebasing anyways.